docs/sources/v0.53.x/using-k6-browser/recommended-practices/hybrid-approach-to-performance.md

```diff
@@ -188,9 +186,8 @@ export async function checkFrontend() {
 
   try {
     await page.goto(BASE_URL);
-    const header = await page.locator('h1').textContent();
-    check(header, {
-      header: (h) => h == 'Looking to break out of your pizza routine?',
+    check(page.locator('h1'), {
```

Comment:
The import should be `import { check } from 'https://jslib.k6.io/k6-utils/1.5.0/index.js';`. This example is still using `k6` `check`.